### PR TITLE
Update UG

### LINKED ISSUE #35: Refactor AB-3 Documentation to StudyTracker
The current documentation for StudyTracker is still AB-3's, hence there is a need to update it. This includes the diagrams in the DG and the discrepancies in the other documentation files.

### PR DESCRIPTION
**Pull request by qreoct**
The UG is a little different from current functionality. 

Let's update it and keep the terminology consistent.

Changes:
- Made command syntax more consistent, with terminology of **command words**, **flags** and **parameters**
- Complete command summary table
- Change pronouns to "you"
- Update command usages
- Update pictures
- Update README
- Update index.md in docs

Partially resolves #35

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -12,58 +15,69 @@ StudyTracker is a **desktop app that lets you save your favourite study spots, o
...
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME* a/ADDRESS`, `a/ADDRESS n/NAME*` is also acceptable.
 
 * If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `a/COM1 a/CLB`, only `a/CLB` will be taken.
+  e.g. if you specify `n/COM1 n/CLB`, only `n/CLB` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
```
Comment (ruoyann) [inline docs/UserGuide.md:83]: For line 86: Can change to 

* `add n/PC Commons a/UTown t/crowded m/wifi r/4` //tag doesn't work with spaces
* `add n/COM2 Basement r/3` (optional arguments are not required) // only name and rating compulsory
* `add n/TR3 a/Yusof Ishak House m/wifi m/aircon` (multiple amenities are acceptable) //changed amenities to those we currently have, and I'm not sure why it's `new` not `add` for the prev ver

Can also add more variations if you want, e.g. multiple tags acceptable

File: docs/UserGuide.md [hunk 2]
```diff
@@ -82,49 +96,48 @@ Some examples of **invalid** user inputs for the *add command* are provided belo
...
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 Tags, amenities and address are all optional.
 </div>
 
-Examples:
+**Examples:**
 ```
 add n/COM1 Basement r/5
 New study spot “COM1 Basement” added to list!
```
Comment (ruoyann) [inline docs/UserGuide.md:120]: Actual output: 
`New study spot added: COM1 Basement; Rating: 5; Operating Hours: -; Address: -; Favourite: false`

It's a little different from the expected output above

File: docs/UserGuide.md [hunk 3]
```diff
@@ -82,49 +96,48 @@ Some examples of **invalid** user inputs for the *add command* are provided belo
...
 
-add n/Starbucks at U-Town r/4 t/noisy m/wifi
-New study spot “Starbucks at U-Town” added to list!
+add n/Starbucks at UTown r/4 t/noisy m/wifi
+New study spot “Starbucks at UTown” added to list!
```
Comment (ruoyann) [inline docs/UserGuide.md:123]: Actual output: 
`New study spot added: Starbucks at UTown; Rating: 4; Operating Hours: -; Address: -; Favourite: false; Tags: [noisy]; Amenities: [wifi]`

File: docs/UserGuide.md [hunk 4]
```diff
@@ -82,49 +96,48 @@ Some examples of **invalid** user inputs for the *add command* are provided belo
...
 
-* Multiple flags can be used at once
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Multiple flags can be used at once.
+</div>
 
-Examples:
+**Examples:**
 ```
 list
```
Comment (ruoyann) [inline docs/UserGuide.md:142]: Not sure how we can display the expected output when it's mostly GUI

E.g. 
when we enter the command `list -t t/noisy`, the actual output to users is `Listed all study spots with Tags: [noisy]`

when we enter the command `list`, actual output: `Listed all study spots`

when we enter the command `list -f`, actual output: `Listed all study spots in Favourites`

File: docs/UserGuide.md [hunk 5]
```diff
@@ -143,22 +156,24 @@ Your study spots are:
...
+* When editing tags and amenities, `t/` or `m/` will append to existing tags and amenities.
+* You can remove a specific tag or amenity with the `rt/` or `rm/` parameter.
+* You can remove all the study spot’s tags by typing `t/` or `m/` without
     specifying any tags after it.
 
-Examples:
+**Examples:**
 ```
 edit spot/tr3 n/Training Room 3
 Study spot “TR3” has been edited to “Training Room 3”
```
Comment (ruoyann) [inline docs/UserGuide.md:179]: actual output: `Edited study spot: Training Room 3; Rating: 4; Operating Hours: -; Address: -; Favourite: false`

File: docs/UserGuide.md [hunk 6]
```diff
@@ -168,9 +183,9 @@ Study spot “TR3” has been edited to “Training Room 3”
...
 Adds a study spot to the StudyTracker's Favourites.
 
-Format: `fav n/NAME*`
+**Format:** `fav n/NAME*`
 
-Examples:
+**Examples:**
 ```
 fav n/COM1 Basement
 Study spot "COM1 Basement" added to favourites!
```
Comment (ruoyann) [inline docs/UserGuide.md:191]: actual output: `Added study spot to favourites: COM1 Basement`

File: docs/UserGuide.md [hunk 7]
```diff
@@ -180,9 +195,9 @@ Study spot "COM1 Basement" added to favourites!
...
 Removes a study spot from the StudyTracker's Favourites.
 
-Format: `unfav n/NAME*`
+**Format:** `unfav n/NAME*`
 
-Examples:
+**Examples:**
 ```
 unfav n/COM1 Basement
 Study spot "COM1 Basement" removed from favourites!
```
Comment (ruoyann) [inline docs/UserGuide.md:203]: actual output: `Removed study spot from favourites: COM1 Basement`

File: docs/UserGuide.md [hunk 8]
```diff
@@ -192,13 +207,13 @@ Study spot "COM1 Basement" removed from favourites!
...
-* Only full words will be matched. e.g. `Han` will not match `Hans`
+* The search is case-insensitive. e.g `library` will match `Library`
+* Only full words will be matched. e.g. `Lib` will not match `Library`
 * Study spots matching at least one keyword will be returned.
 
-Examples:
+**Examples:**
 ```
 find library
 Found the following study spots matching “library”:
```
Comment (ruoyann) [inline docs/UserGuide.md:219]: Actual output: `2 study spot(s) listed!`

File: docs/UserGuide.md [hunk 9]
```diff
@@ -211,12 +226,11 @@ Found the following study spots matching “library”:
...
+**Format:** `delete n/NAME*`
 
 * Deletes the study spot with the specified **name**.
 
-
-Examples:
+**Examples:**
 ```
 delete n/COM1
 COM1 has been deleted.
```
Comment (ruoyann) [inline docs/UserGuide.md:236]: actual output: `Deleted study spot: Central Library; Rating: 3; Operating Hours: Weekdays: 0900-2200, Weekends: 0900-2200; Address: Next to FASS; Favourite: false; Tags: [cold][quiet]; Amenities: [wifi][charger][aircon]`

^ just a random studyspot i deleted

File: docs/UserGuide.md [hunk 10]
```diff
@@ -230,9 +244,9 @@ Clears all entries from the StudyTracker.
...
 </div>
 
-Format: `clear`
+**Format:** `clear`
 
-Example:
+**Example:**
 ```
 clear
 All study spots have been cleared.
```
Comment (ruoyann) [inline docs/UserGuide.md:252]: actual output: `StudyTracker has been cleared!`

File: docs/UserGuide.md [hunk 11]
```diff
@@ -242,67 +256,73 @@ All study spots have been cleared.
...
 Closes the StudyTracker.
 
-Format: `exit`
+**Format:** `exit`
 
-Example:
+**Example:**
 ```
 exit
 Goodbye!
```
Comment (ruoyann) [inline docs/UserGuide.md:264]: It doesn't say goodbye, quits straightaway haha unless it's too fast

File: docs/UserGuide.md [hunk 12]
```diff
@@ -242,67 +256,73 @@ All study spots have been cleared.
...
+### Removing command aliases: `unalias`
+
+Removes user-defined aliases.
+
+**Format** `unalias al/ALIAS*` to remove alias
+
+**Example:**
+```
+unalias al/myAdd
+Removed alias 'myAdd'.
```
Comment (ruoyann) [inline docs/UserGuide.md:304]: actual output: `Removed alias myAdd.`

File: docs/UserGuide.md [hunk 13]
```diff
@@ -311,11 +331,13 @@ _Details coming soon ..._
 Action | Format, Examples
 --------|------------------
 **Add** | `add n/NAME* r/RATING* a/ADDRESS t/TAG... m/AMENITY...` <br> e.g., `add n/COM1 r/5`
+**Alias** | `alias al/ALIAS* cmd/COMMAND*`<br> e.g., `alias al/home cmd/find home`
```
Comment (ruoyann) [inline docs/UserGuide.md:334]: typo of e.g. (extra comma)

File: docs/UserGuide.md [hunk 14]
```diff
@@ -311,11 +331,13 @@ _Details coming soon ..._
 Action | Format, Examples
 --------|------------------
 **Add** | `add n/NAME* r/RATING* a/ADDRESS t/TAG... m/AMENITY...` <br> e.g., `add n/COM1 r/5`
```
Comment (ruoyann) [inline docs/UserGuide.md:333]: typo of e.g. (extra comma)

File: docs/UserGuide.md [hunk 15]
```diff
@@ -311,11 +331,13 @@ _Details coming soon ..._
...
+**Alias** | `alias al/ALIAS* cmd/COMMAND*`<br> e.g., `alias al/home cmd/find home`
 **Clear** | `clear`
 **Delete** | `delete n/NAME*` <br> e.g., `delete n/COM1`
```
Comment (ruoyann) [inline docs/UserGuide.md:336]: typo of e.g. (extra comma)

File: docs/UserGuide.md [hunk 16]
```diff
@@ -311,11 +331,13 @@ _Details coming soon ..._
...
 **Edit** | `edit n/NAME* n/NEW_NAME a/NEW_ADDRESS t/NEW_TAG m/NEW_AMENITY r/NEW_RATING`<br> e.g.,`edit n/tr3 n/Training Room 3`
```
Comment (ruoyann) [inline docs/UserGuide.md:337]: typo of e.g. (extra comma)

File: docs/UserGuide.md [hunk 17]
```diff
@@ -311,11 +331,13 @@ _Details coming soon ..._
...
 **Favourite**  |  `fav n/NAME*` <br> e.g. `fav n/COM1`
 **Find** | `find n/NAME*`<br> e.g., `find n/lib`
```
Comment (ruoyann) [inline docs/UserGuide.md:339]: typo of e.g. (extra comma)

File: docs/index.md [hunk 18]
```diff
@@ -5,16 +5,18 @@ title: StudyTracker
...
+[![latest release](https://badgen.net/github/release/AY2122S1-CS2103T-T09-1/tp)](https://github.com/AY2122S1-CS2103T-T09-1/tp/releases)
 
 ![Ui](images/Ui.png)
 
-**StudyTracker is a desktop application that lets you save your favourite study spots, optimized for use via Command Line Interface (CLI).**
-While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**StudyTracker is a desktop application that maximises your productivity by making it easy to find the best places to study.**
+
+StudyTracker is optimized for use via Command Line Interface (CLI).
+Easy to learn, yet powerful when mastered with various customizations.
```
Comment (ruoyann) [inline docs/index.md:16]: yay nice work jeff!

**Review by mslevis — APPROVED** [review]
Great work! GUI Features Section look really good

**Pull request merged**